src/graphcast_agent/waku_handling.rs

```diff
@@ -192,6 +182,8 @@ fn node_config(
         gossipsub_params: Some(gossipsub_params),
         dns4_domain_name: None,
         websocket_params: None,
+        dns_discovery_urls: vec![],
```

Comment:
should these 2 params be passed in from the radio since we have those configs iirc?